script.js

```diff
@@ -0,0 +1,19 @@
+const hamburguer = document.querySelector('.menu');
+const navMenu = document.querySelector('.list');
+const icons = document.querySelectorAll('.menu-icon');
+const hamburguerIcon = document.querySelector('#hamburguer');
+
+hamburguer.addEventListener('click', () => {
+  navMenu.classList.toggle('active');
+  icons.forEach((icon) => {
+    icon.classList.toggle('close-menu');
+  });
+});
```

Comment:
- [ ] Please make sure that the menu element stays intact even after scrolling down. Currently, as the user scrolls down with the menu active, the top section of the menu stays in the header, leaving a small empty space at the top and losing the icon to close the menu. You could add a condition that adds the property `overflow: hidden` to the body when the menu is active or fix the menu itself, please make sure to address this issue.

![](https://i.imgur.com/Ihr3WPZ.png)